File: Valid Palindrome.py
class Solution(object):
    def isPalindrome(self, s):
        """
        :type s: str
        :rtype: bool
        """
        s = s.lower()
        start = 0
        end = len(s)-1
        for i in range(0,len(s)):
            if s[start].isalnum() == False:
                start+=1
                continue
            if s[end].isalnum() == False:
                end-=1
                continue
            if s[start] != s[end]:
                return False
            else:
                start += 1
                end  -= 1
        return True

                
    
        
        

File: php/Valid Palindrome.php
<?php

class Solution {
function isPalindrome($s) {
    $s = preg_replace("/[^a-z0-9]/","",strtolower($s));
    $end = strlen($s) -1;
    echo $s;
    for ($start = 0;$start <= $end; $start++,$end--){
        if ($s[$start] != $s[$end]){
            return false;
        }
    }
    return true;
}
}